src/notebook/cells/widget.ts

```diff
@@ -430,6 +430,7 @@ class CodeCellWidget extends BaseCellWidget {
   execute(kernel: IKernel): Promise<KernelMessage.IExecuteReplyMsg> {
     let model = this.model as ICodeCellModel;
     let code = model.source;
+    model.outputs.clear();
```

Comment:
Isn't this already being run in the output area execution method, at https://github.com/jupyter/jupyterlab/pull/656/files#diff-1eb8c480985ee63765746a78fa427fedR165?